src/pkg/utils/random.go

```diff
@@ -42,6 +43,10 @@ func First30last30(s string) string {
 
 // ColorWrap changes a string to an ansi color code and appends the default color to the end
 // preventing future characters from taking on the given color
+// returns string as normal if color is disabled
 func ColorWrap(str string, attr color.Attribute) string {
+	if config.NoColor {
```

Comment:
First30Last30 and ColorWrap should probably be in `message instead` - `random` isn't junk drawer random - also message could be refactored out of RandomString and `random.go` could be moved to `helpers`